src/workerd/io/BUILD.bazel

```diff
@@ -52,6 +52,7 @@ wd_cc_library(
         "io-util.c++",
         "tracer.c++",
         "worker.c++",
+        "worker-fs.c++",
```

Comment:
Review note: for the time being I won't be trying to separate this out into a separate dep. That's something that can be done later in a separate PR